source/index.txt

```diff
@@ -41,23 +41,22 @@ Once you have installed the |mdb-shell| and added it to your system
 The |mdb-shell| versus the Legacy ``mongo`` Shell
 -------------------------------------------------
 
-The new MongoDB Shell, ``mongosh``, offers numerous
-advantages over the legacy :binary:`~bin.mongo` shell, such as:
-
-- Improved syntax highlighting.
-
-- Improved command history.
-
-- Improved logging.
-
-Currently ``mongosh`` supports a subset of the :binary:`~bin.mongo`
-shell methods. Achieving feature parity between ``mongosh`` and the
-``mongo`` shell is an ongoing effort.
-
-To maintain backwards compatibility, the methods that ``mongosh``
-supports use the same syntax as the corresponding methods in the
-:binary:`~bin.mongo` shell. To see the complete list of methods
-supported by ``mongosh``, see
+The legacy ``mongo`` shell was deprecated in MongoDB 5.0 and removed in
+MongoDB 6.0. The new MongoDB Shell, ``mongosh``, offers numerous
+advantages over the legacy shell. The new shell has improved:
+
+- compatibility with the MongoDB `Node.js
+  <https://www.mongodb.com/docs/drivers/node/current/>`__ driver
+- syntax highlighting
+- command history
+- logging
+
```

Comment:
Removed the "Currently working to achieve parity" language. That was from the early days of the project. I'll confirm in external review that it is no longer needed.